nanopc-t6.mk

```diff
@@ -132,12 +134,16 @@ u-boot-proper: $(TPL_BIN) $(UBOOT_PATH)/.config u-boot-config
 u-boot-apply-bootcmd:
 	cp $(BOOT_CMD) $(UBOOT_HEADER)
 
+.PHONY: u-boot-apply-fit-args
+u-boot-apply-fit-args:
+	cp $(FIT_ARGS) $(UBOOT_FIT_ARGS)
```

Comment:
I see that you re using the same pattern as `u-boot-apply-bootcmd`, but I think it's a bad pattern. The good one would be:
```make
$(U_BOOT_FIT_ARGS): $(FIT_ARGS)
	cp $< $@
```